### PR TITLE
Feat: add docker build of fate_test

### PR DESCRIPTION
**Pull request by owlet42**
**Changes**
- Add docker build of fate_test
- Add fate_test test case for KubeFATE

**Test**
image

### CODE REVIEW

File: k8s-deploy/pkg/modules/helm_chart.go [hunk 1]
```diff
@@ -1,67 +0,0 @@
-/*
```
Comment (JingChen23) [inline k8s-deploy/pkg/modules/helm_chart.go:1]: Why do we remove this file?

File: docker-build/README.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ This contains the builds of some images for KubeFATE to deploy FATE.
 - nginx
 - spark
 - python-spark
+- fate-test
```
Comment (JingChen23) [inline docker-build/README.md:9]: Why this 4 lines have different indents?

Comment (JingChen23) [reply docker-build/README.md:9]: I meam different with client on line 5

File: k8s-deploy/pkg/modules/helm_chart.go (hunk 1 above)
Comment (owlet42) [reply k8s-deploy/pkg/modules/helm_chart.go]: It was a mistake, it has been recovered

File: docker-build/README.md (hunk 2 above)
Comment (owlet42) [reply docker-build/README.md]: This images list, I have modified.

**Review by JingChen23 — APPROVED** [review]

**Pull request merged**